src/app/core/services/mocks/MOCK_RULE_RESPONSE.ts

```diff
@@ -1,32 +1,31 @@
 import { RuleResponse } from 'src/app/models/rule-response';
 
 export const MOCK_RULE_RESPONSE: RuleResponse = {
-  rule: `
-// Copyright (c) 2017 Datical, Inc. All Rights Reserved.
-/*
-@author Laurent Rochette
-@version 1.0
-@date July 19, 2017
-@description Error if GRANT is detected
-*/
+  rule: `// Copyright (c) 2020 Datical, Inc. All Rights Reserved.
```

Comment:
```suggestion
  rule: `
// Copyright (c) 2020 Datical, Inc. All Rights Reserved.
```

Reply:
if so, i think it might be fixed with latest release at https://tabuckner.github.io/rules-wizard-ui/wizard, can you double check please?